packages/teams-js/src/public/storage.ts

```diff
@@ -0,0 +1,24 @@
+import { GlobalVars } from '../internal/globalVars';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { HostClientType } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Contains functionality to allow web apps to store data in webview cache
+ *
+ * @beta
+ */
+export namespace storage {
+  export function isWebStorageClearedOnUserLogOut(): boolean {
```

Comment:
Please add reference documentation comments indicating what this function does.

Reply:
done